src/components/Navigation/styles.js

```diff
@@ -19,7 +19,7 @@ export const StyledMenu = styled.ul`
   list-style: none;
   overflow: hidden;
   transition: height 0.5s;
-  height: ${props => (props.open ? '306px' : '0')};
+  height: ${props => (props.open ? '357px' : '0')};
```

Comment:
I guess it should be 'auto'. We don't know the final size anyways

Reply:
I agree. Need to fix.

Reply:
But, If we use 'auto' browser don't know the heigth of element and there will be not smooth transition in height.

Reply:
Do we need this smooth transition at all :)?  we'll see after we have final designs and discuss UX within the team

Reply:
Maybe for today it is not necessary. I can set 'auto' for convenience if somebody decide to add menu item and not change height each time.

Reply:
Nice!